compose/wordpress.md

```diff
@@ -0,0 +1,145 @@
+---
+description: Getting started with Compose and WordPress
+keywords: documentation, docs,  docker, compose, orchestration, containers
+title: "Quickstart: Compose and WordPress"
+---
+
+You can use Docker Compose to easily run WordPress in an isolated environment
+built with Docker containers. This quick-start guide demonstrates how to use
+Compose to set up and run WordPress. Before starting, you'll need to have
+[Compose installed](install.md).
+
+### Define the project
+
+1. Create an empty project directory.
+
+    You can name the directory something easy for you to remember.
+    This directory is the context for your application image. The
+    directory should only contain resources to build that image.
+
+    This project directory will contain a `docker-compose.yml` file which will
+    be complete in itself for a good starter wordpress project.
+
+    >**Tip**: You can use either a `.yml` or `.yaml` extension for
+    this file. They both work.
+
+2. Change directories into your project directory.
+
+    For example, if you named your directory `my_wordpress`:
+
+        cd my_wordpress/
```

Comment:
The entire set of Compose docs is done without the yaml tag on the code fences, which means that the code samples don’t look red. I think if we are going to do this, we should do it in one sweeping change otherwise it’s going to look weird, so skipping for now.